### PR TITLE
refactored to mina_intf, mina_debug, mina_incremental)

### PR DESCRIPTION
**Pull request by Kida007**
This PR : 
- refactored to mina_intf, mina_debug, mina_incremental)
Explain how you tested your changes here.

Tested :
- make build

Checklist:

- [ ] Document code purpose, how to use it
  - Mention expected invariants, implicit constraints
- [ ] Tests were added for the new behavior
  - Document test purpose, significance of failures
  - Test names should reflect their purpose
- [ ] All tests pass (CI will check this if you didn't)
- [ ] Serialized types are in stable-versioned modules
- [ ] Does this close issues? List them:

### CODE REVIEW

**Review by bkase — COMMENTED** [review]
Can you undo the coda automation part?

**Comment by bkase** [comment]
!ci-build-me

File: scripts/Brewfile.lock.json [hunk 1]
```diff
@@ -31,55 +31,43 @@
         "bottle": false
```
Comment (bkase) [inline scripts/Brewfile.lock.json:31]: can you remove this file?

**Review by bkase — APPROVED** [review]

**Comment by bkase** [comment]
!ci-build-me

**Comment by Kida007** [comment]
ci looks good, can this be landed @bkase

**Comment by mrmr1993** [comment]
!ci-build-me

**Review by mrmr1993 — APPROVED** [review]

**Pull request merged**